zScanner/DomainModel/PageDomainModel.swift

```diff
@@ -13,6 +13,7 @@ struct PageDomainModel: Equatable {
     var index: Int
     var correlationId: String
     var relativePath: String
+//    var description: String
```

Comment:
Generally, don't commit commented code. 
You can add the code here and put empty string somewhere, you can commit `// TODO:`. but don't commit commented code without any other comment when to uncomment it or why it is there.